nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -184,11 +196,17 @@ public void testExportWithParams() throws Exception {
         // Check header
         List<String> header = Arrays.asList(lines.get(0).split(","));
         assertArrayEquals(
-                new String[] { "dc:contributors", "dc:coverage", "dc:created", "dc:creator", "dc:description",
-                        "dc:expired", "dc:format", "dc:issued", "dc:language", "dc:lastContributor", "dc:modified",
-                        "dc:nature", "dc:publisher", "dc:rights", "dc:source", "dc:subjects", "dc:title", "dc:valid",
-                        "cpx:complex/foo" },
-                header.subList(18, 37).toArray());
+                new String[] { "dc:contributors", "dc:coverage", "dc:coverage[label]", "dc:created", "dc:creator",
+                        "dc:description", "dc:expired", "dc:format", "dc:issued", "dc:language", "dc:lastContributor",
+                        "dc:modified", "dc:nature", "dc:nature[label]", "dc:publisher", "dc:rights", "dc:source",
+                        "dc:subjects", "dc:subjects[label]", "dc:title", "dc:valid", "cpx:complex/foo" },
+                header.subList(18, 40).toArray());
+
+        List<String> content = lines.subList(1, lines.size());
+        for (String doc : content) {
+            List<String> properties = Arrays.asList(doc.split(","));
+            assertTrue(properties.contains("Article FR"));
```

Comment:
What do you think about:
```suggestion
            assertTrue(doc.contains(",Article FR,"));
```
?

Reply:
What if it's first or last?